index.html

```diff
@@ -107,14 +92,14 @@ <h3 class="title">Our Mission</h3>
       </div>
     </div>
 
-    <section class="hero2">
+    <section class="gallery">
       <div class="overlay">
-        <div class="sheep-photo-above">
-          <img src="img/sheep_above.png" alt="" class="sheep_above" />
+        <div class="sheep__photo__above">
```

Comment:
nu e ok sa folosesti asa cu 3 nume aici
```suggestion
        <div class="gallery__sheep">
```